src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/Sprint.kt

```diff
@@ -60,7 +61,13 @@ object Sprint : Module("Sprint", ModuleCategory.MOVEMENT, gameDetecting = false)
     fun correctSprintState(movementInput: MovementInput, isUsingItem: Boolean) {
         val player = mc.thePlayer ?: return
 
-        if (onlyOnSprintPress && !player.isSprinting && !mc.gameSettings.keyBindSprint.isKeyDown)
+        val module = SuperKnockback
+        if (module.breakSprint()) {
```

Comment:
if (SuperKnockback.breakSprint())

Reply:
> if (SuperKnockback.breakSprint())

This could be applied to many many many many things like
```
        val module = Blink
        if (module.blinkingSend() || mc.thePlayer.isDead)
```
but it's not an issue, I don't feel like changing this everywhere